app/assets/stylesheets/views/_taxons.scss

```diff
@@ -69,8 +69,8 @@
 }
 
 .taxon-page__see-more {
-  display: block;
-  margin-top: govuk-spacing(6);
+  position: relative;
+  top: govuk-spacing(3);
```

Comment:
I think I understand this change but is it worth these lines of code? Removing them only moves it up a little bit, I don't see any problem with that particularly.